CHANGELOG.md

```diff
@@ -12,9 +12,8 @@ x.x.x
 - Add support for attaching existing FSx for Ontap and FSx for OpenZFS File Systems.
 - Add support for FSx Lustre Persistent_2 deployment type.
 - Show `requested_value` and `current_value` values in the change set when adding or removing a section.
-- Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to configure memory-based
-  scheduling in Slurm.
-  - Add new configuration parameter to override default value of schedulable memory on compute nodes.
+- Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to configure memory-based scheduling in Slurm.
+- Add new configuration parameter `Scheduling/SlurmQueues/ComputeResources/SchedulableMemory` to override default value of schedulable memory on compute nodes.
```

Comment:
of schedulable memory --> of the compute nodes memory seen by the scheduler. 

What do you think?

Reply:
Id' say `memory seen by the scheduler on compute nodes.`